### PR TITLE
Adscript

### PR DESCRIPTION
**Pull request by wjoosen**

### CODE REVIEW

**Review by georgechoustoulakis — APPROVED** [review]

File: adscript/README.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# adscript-connector-web
+
+The AdScript connector provides a AdScript integration for THEOplayer.
```
Comment (GillesMoris-THEO) [inline adscript/README.md:3]: Nitpick: `an AdScript integration`

File: adscript/README.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+```javascript
+adScriptConnector.updateMetadata(newMetadata);
+```
+
+## Updating userInfo
+
+If the user info has changed during playback, you can update it with:
+
+```javascript
+adScriptConnector.updateUser(i12n);
```
Comment (GillesMoris-THEO) [inline adscript/README.md:99]: I think `updateUser` and `updateMetadata` above were changed to `setI12n` and `setContentMetadata` respectively, update here.

Comment (GillesMoris-THEO) [reply adscript/README.md:99]: + Do we need to document usage of `setPlayerState`?

Comment (GillesMoris-THEO) [inline adscript/README.md:99]: `i12n` is a bit of a vague parameter name for a method called `updateUser` right? Can we make it more sensible or is this just weird AdScript naming?

Comment (georgechoustoulakis) [reply adscript/README.md:99]: The naming comes directly from the AdScript docs

**Pull request merged**